docs/security.rst

```diff
@@ -30,3 +30,29 @@ either with `kafkastore.sasl.kerberos.service.name` or in the JAAS file.
 
 If the Schema Registry has a different service name than Kafka, at this time `zookeeper.set.acl` must be set to `false`
 in both the Schema Registry and Kafka.
+
+Upgrade from HTTP to HTTPS for REST API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default Schema Registry ships with configs that allow you to make REST API calls over HTTP. If you would want to upgrade to HTTPS in an existing cluster, you need to perform the outlined steps:
+
+- Add/Modify the ``listeners`` config  to include HTTPS. For example: http://0.0.0.0:8081,https://0.0.0.0:8082
+- Configure the appropriate SSL configurations
+
+    - ``ssl.keystore.`` configs to setup keystore related configs
+    - ``ssl.truststore.`` configs to setup truststore related configs (required only when ``ssl.client.auth`` set to true)
+   
+- Do a rolling bounce of the cluster
+
+By default, all communications(forwarding from slave to master for schema register) between the
```

Comment:
If they're enabling SSL, they almost definitely want to disable plain http. I would write this to assume this -- maybe explain that this process enables https, but still defaults to http so schema registry instances can still communicate before all nodes have been restarted. They will continue to use http as the default until configured not to. To switch to https as the default and disable http support, perform the following steps:

Reply:
This might also be clearer if you introduce the two settings up front before walking through the process.